### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/debugging.md`

### PR DESCRIPTION
**Pull request by rafaelem2**
In this pull request I traslated the `docs/tutorial/debugging.md` to portguese.
I'm open to feedbacks, let me know if it needs any changes.

### CODE REVIEW

**Comment by rafaelem2** [comment]
@Kludex Can you check my translation please?

**Comment by guishas** [comment]
I am from Brazil and I think this translation is fine. 👍

File: docs/pt/docs/tutorial/debugging.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+mas não é chamado quando outro arquivo o importa, como em:
+
+```Python
+from myapp import app
+```
+
+#### Mais detalhes
+
+Digamos que o seu arquivo tem o nome de `myapp.py`.
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/debugging.md:33]: A little suggestion:

```suggestion
Digamos que o seu arquivo se chama `myapp.py`.
```

File: docs/pt/docs/tutorial/debugging.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+<div class="termy">
+
+```console
+$ python myapp.py
+```
+
+</div>
+
+a variável interna `__name__` no seu arquivo, criada automaticamente pelo Python, vai ter como valor a string `"__main__"`.
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/debugging.md:45]: I believe it's missing something to connect two sentences here - maybe you forgot from the original sentence?

```suggestion
então, a variável interna `__name__` no seu arquivo, criada automaticamente pelo Python, vai ter como valor a string `"__main__"`.
```

File: docs/pt/docs/tutorial/debugging.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+Então, a linha:
+
+```Python
+   uvicorn.run(app, host="0.0.0.0", port=8000)
+```
+
+não será executada.
+
+!!! info "Informação"
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/debugging.md:77]: I believe the `"Informação"` isn't really necessary here. It's adding nothing - actually, it isn't even in the original language, and it's redundant. 🤔

```suggestion
!!! info
```

File: docs/pt/docs/tutorial/debugging.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   Para mais informações, olhe <a href="https://docs.python.org/3/library/__main__.html" class="external-link" target="_blank">a documentação oficial do Python</a>.
+
+## Rode o seu código com o seu debugger
+
+Por conta de você estar rodando o servidor Uvicorn diretamente do seu código, você pode chamar seu programa Python (a sua aplicação FastAPI) diretamente do debugger.
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/debugging.md:82]: I believe the reading will be better without the article, what do you think?

```suggestion
Por conta de você estar rodando o servidor Uvicorn diretamente do seu código, você pode chamar seu programa Python (sua aplicação FastAPI) diretamente do debugger.
```

**Review by peidrao — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit ac420db1a2fd5159175bd1dbd2e5890fd32ecb35 at: https://649a1e9c1636a41623534cfe--fastapi.netlify.app

**Pull request closed without merging**